3.2/root/usr/share/container-scripts/mongodb/init-petset-replset.sh

```diff
@@ -118,31 +70,28 @@ function add_member() {
   local replset_addr
   replset_addr="${MONGODB_REPLICA_NAME}/${endpoints}"
 
-  if ! mongo admin -u admin -p "${MONGODB_ADMIN_PASSWORD}" --host "${replset_addr}" --eval "${script}" --quiet; then
+  if ! mongo admin -u admin -p "${MONGODB_ADMIN_PASSWORD}" --host "${replset_addr}" --eval "while (!rs.add('${host}').ok) { sleep(100); }" --quiet; then
```

Comment:
By the way: I figured out that this change is necessary for #224 (because Docker does not support SRV DNS records).

As we started to talk about this: @jim-minter Do you think this change might cause some troubles?